ovs/lib/albacontroller.py

```diff
@@ -285,6 +287,13 @@ def remove_cluster(alba_backend_guid):
         if len(albabackend.asds) > 0:
             raise RuntimeError('A backend with claimed OSDs cannot be removed')
 
+        for abm_service in albabackend.abm_services:
+            asd_node = AlbaNodeList.get_albanode_by_ip(abm_service.service.storagerouter.ip)
+            try:
+                asd_node.client.list_maintenance_services()
+            except requests.exceptions.ConnectionError as ce:
+                raise RuntimeError('Node {0} is not reachable, backend cannot be removed. {1}'.format(asd_node.ip, ce))
```

Comment:
This only covers storagenodes. If you run a non-hyperconverged setup, it won't check the ovs-cluster member nodes. For this, you still need the same logic as in the arakoon checkup during add_cluster.